pkg/registry/apis/provisioning/resources/resources.go

```diff
@@ -160,8 +161,14 @@ func (r *ResourcesManager) WriteResourceFromFile(ctx context.Context, path strin
 	parsed.Meta.SetUID("")             // clear identifiers
 	parsed.Meta.SetResourceVersion("") // clear identifiers
 
-	// Update will also create (for resources we care about)
 	_, err = parsed.Client.Update(ctx, parsed.Obj, metav1.UpdateOptions{})
+	if err != nil && apierrors.IsAlreadyExists(err) {
```

Comment:
should this be `IfNotFound`

or invert and do Create.... IfAlreadyExists... Update?

Reply:
Oh yeah, good catch, thanks!